ansible/upgrade_sonic.yml

```diff
@@ -77,7 +77,7 @@
         port: 22
         state: started
         search_regex: "OpenSSH"
-        delay: 30
+        delay: 180
```

Comment:
I would like to suggest an improvement here.
i would keep the 30 seconds but will add 6 retries which will lead to the 3 minutes you mentioned 
if it is up before the 180 seconds we dont need to wait.

Reply:
The problem here is that we can't probe too early, because the device may be updating firmware and still accessible via SSH. In that case, script may consider reboot is already done before system goes down for rebooting. A 180 seconds delay is a reasonable value for system to complete firmware updating and go down for rebooting.